demos/demo.html

```diff
@@ -1,19 +1,28 @@
-<div class="o-grid-container o-grid-container--snappy">
-	<div class="o-grid-row">
-		<h1>n-newsletter-signup demo</h1>
-	</div>
-	<div class="o-grid-row">
-		<div data-o-grid-colspan="12 S6 M4 L3">
-			{{> n-newsletter-signup/templates/newsletter flags=@root.flags}}
-		</div>
-	</div>
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>demo</title>
+		<link rel="stylesheet" href="/public/main.css">
```

Comment:
@magsallen I would like to chat about this. There's no manifest.json file being used in this repo. and I think I will need to do this on every single html file for the CSS to work.